central/tlsconfig/tlsconfig_test.go

```diff
@@ -30,11 +31,11 @@ func (s *tlsConfigTestSuite) TestGetAdditionalCAs() {
 
 	additionalCAs, err := GetAdditionalCAs()
 	s.Require().NoError(err)
-	s.Require().Len(additionalCAs, 3, "Could not decode all certs")
+	s.Require().Len(additionalCAs, 4, "Could not decode all certs")
 	s.True(s.isCommonNameInCerts(additionalCAs, "CENTRAL_SERVICE: Central"), "Could not find cert from multiple crt file")
 
-	// non .crt files should be ignored
-	s.FileExists("testdata/no_ca_cert.pem")
+	// non .crt or .pem files should be ignored
```

Comment:
You've invested so much into making the extensions dynamic, but the comment is not dynamically generated from the extensions list :p

Reply:
lol, I don't think I'll add a `go generate` to update this comment ;)